bedrock/firefox/templates/firefox/new/base_download.html

```diff
@@ -0,0 +1,119 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+{% from "macros.html" import fxa_link_fragment, fxa_email_form with context %}
+{% from "macros-protocol.html" import picto_card with context %}
+
+{% add_lang_files "firefox/new/quantum" %}
```

Comment:
Reorganizing these files is a worthwhile effort for simplification, but I think mixing .lang files and templates like this could also risk making things more complicated than having separate templates.

For example, this file adds `firefox/new/quantum.lang`, yet others below go on to extend it addng `firefox/new/trailhead.lang`. Which .lang file then dictates activation? (I _think_ it would be the former?). And when making a change to a string, which .lang file should be updated? It feels like this could end up being complicated. Thoughts?

Reply:
I think it would be useful to figure out an uncomplicated solution for this if we were not in the process of migrating to a totally different l10n system.

As it is I have reverted the trailhead files so there is no longer any lang file overlap.